server/sonar-server/src/main/java/org/sonar/server/issue/index/IssueDoc.java

```diff
@@ -215,11 +225,21 @@ public boolean isNew() {
     throw new IllegalStateException("isNew is only available for batch");
   }
 
+  /**
+   * @deprecated since 5.5, replaced by {@link #effort()}
+   */
   @Override
   @CheckForNull
+  @Deprecated
   public Duration debt() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)